uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/rules/PowerLiftIncidentRule.java

```diff
@@ -32,30 +33,39 @@ public void evaluate() throws Throwable {
                 try {
                     base.evaluate();
                 } catch (final Throwable originalThrowable) {
-                    String powerLiftIncidentDetails = null;
-                    try {
-                        Logger.e(
-                                TAG,
-                                "Encountered error during test....creating PowerLift incident.",
-                                originalThrowable
-                        );
-                        powerLiftIncidentDetails = powerLiftIntegratedApp.createPowerLiftIncident();
-                    } catch (final Throwable powerLiftError) {
-                        Logger.e(
+                    if (BuildConfig.SEND_POWERLIFT_FAILURES) {
```

Comment:
Rather than putting this condition here, the better way to do would be to just not add the rule to RuleChain by default and move the condition in the RulesHelper and only add the powerlift incident rule if  `if (BuildConfig.SEND_POWERLIFT_FAILURES)`

Reply:
Makes sense!